### PR TITLE
Use HA specific content on About and Legal screens

### PR DESCRIPTION
**Pull request by devinjameson**
Why: prior to this commit the About and Legal screens were using placeholder content.

Co-Authored-By: Alex Chen <achen178@gmail.com>

aboutlegal

### CODE REVIEW

File: src/More/About.tsx [hunk 1]
```diff
@@ -18,9 +12,11 @@ export const AboutScreen: FunctionComponent = () => {
   const { t } = useTranslation()
 
   const osInfo = `${Platform.OS} v${Platform.Version}`
-  const pathCheckWebAddress = "pathcheck.org"
-  const pathCheckUrl = "https://pathcheck.org/"
   const { applicationName, versionInfo } = useApplicationInfo()
+  const healthAuthorityName =
```
Comment (devinjameson) [inline src/More/About.tsx:16]: @aledustet Will all this stuff be wrapped up in the config context? It's starting to feel pretty brittle.

Comment (aledustet) [reply src/More/About.tsx:16]: Definitely, this all os a candidate, for this screen however there might be changes coming soon but we should have this done soon.

File: src/More/Licenses.tsx [hunk 2]
```diff
@@ -20,12 +20,16 @@ const Licenses: FunctionComponent = () => {
...
-  const pathCheckUrl = "https://pathcheck.org/"
   const privacyPolicyUrl =
     env.PRIVACY_POLICY_URL || "https://pathcheck.org/privacy-policy/"
+  const healthAuthorityName =
+    env.GAEN_AUTHORITY_NAME || "PathCheck Organization"
+  const haAddressLine1 = env.GAEN_AUTHORITY_ADDRESS_LINE_1 || "58 Day Street"
+  const haAddressLine2 = env.GAEN_AUTHORITY_ADDRESS_LINE_2 || "Box 441621"
+  const haAddressLine3 =
+    env.GAEN_AUTHORITY_ADDRESS_LINE_3 || "Somerville, MA 02144"
+  const haAddressLine4 = env.GAEN_AUTHORITY_ADDRESS_LINE_4 || "USA"
```
Comment (aledustet) [inline src/More/Licenses.tsx:31]: I'm kind of worried about this but we will not solve international addresses when the moment comes that a different country won't have all the components we have here. Probably this ship sailed but did we consider having one address with `\n` to format it?

Comment (devinjameson) [reply src/More/Licenses.tsx:31]: Yeah we're actually removing all this stuff anyways but that's a good point!

**Pull request merged**